nntrainer/utils/nntr_threads.cpp

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2022 Jjioong Moon <jijoong.moon@samsung.com>
+ *
+ * @file nntr_threads.cpp
+ * @date 07 July 2022
+ * @brief Thread Management for NNTrainer
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <nntr_threads.h>
+
+namespace nntrainer {
+
+ParallelBatch::ParallelBatch(threaded_cb threaded_cb, unsigned int batch_size,
+                             void *user_data_) :
+  cb(threaded_cb),
+  batch(batch_size),
+  num_workers(NNTR_NUM_THREADS),
```

Comment:
Instead of the code at the line 22,
```suggestion
  num_workers(NNTR_NUM_THREADS > batch ? 1 : NNTR_NUM_THREADS ),
```

Reply:
fixed. Thanks.